### PR TITLE
Fix input data from i2s_bidi_slave program

### PR DESCRIPTION
**Pull request by murdo-g**
Thanks for sharing this great example. 

I was having issues using the **i2s_bidi_slave** program with an SGTL5000 codec. Input data appeared to be bit shifted resulting in heavy distortion.

I restructured the PIO program to more closely resemble the **i2s_in_slave** program which fixed the issue in my case and thought I'd share it.

From the sounds of things this is the same issue users mentioned in #6.

### CODE REVIEW

File: i2s.pio [hunk 1]
```diff
@@ -88,35 +88,37 @@ dataR:
...
-  jmp start_l                   ;
-continue_r:
-  out pins 1                    ;
   wait 1 pin 1                  ;
   in pins 1                     ;
-  jmp loop_r                    ;
+  wait 0 pin 1                  ;
+  out pins 1                    ; update DOUT
+  jmp pin loop_r                ; if LRCK is still high, we're still sampling this word
+  jmp start_l                   ; otherwise, start the loop over
```
Comment (malacalypse) [inline i2s.pio:120]: Please delete this last jump, it's unnecessary.

**Review by malacalypse — CHANGES_REQUESTED** [review]
I'll be happy to accept this after the last line is removed. Please leave the comment (and line up the semicolon with the line above it, something like:

```
  jmp pin loop_r                ; if LRCK is still high, we're still sampling this word
                                ; implicit jmp to start_l: otherwise, start the loop over
```

Would be nice. If you want to do the same thing at line 158 (last line of the audio input PIO) that'd be cool too.  Thank you!

**Comment by murdo-g** [comment]
Removed the final jump instructions for both **i2s_bidi_slave** and **i2s_in_slave**

**Review by malacalypse — APPROVED** [review]

**Comment by malacalypse** [comment]
Thanks @murdo-g for taking the time to do that.

**Pull request merged**